### DISCUSSION TITLE
Get path to the current project

### DISCUSSION THREAD

**Discussion by Martyx00** [NationalSecurityAgency/ghidra#2833] [Q&A] [answered]
Hello,
Is there any API function that would return file system path to the current project file (.gpr)?

Thanks!

**Comment by ryanmkurtz** [comment, marked as answer]
I haven't tested this but you can try:

`AppInfo.getActiveProject().getProjectLocator().getMarkerFile()`

**Reply by Martyx00** [reply]
Yes, thanks!
